_config/debugbar.yml

```diff
@@ -21,7 +21,9 @@ DebugBar:
   # Maximum number of queries to display
   query_limit: 200
   # Number of DB queries before a warning will be displayed
-  warn_query_limit: 100
+  warn_query_limit: 10
+  # The link to show for large numbers of database queries, and other performance issues
```

Comment:
Your description is a bit confusing

I'd say:
"Link to show when there are notable performance issues, e.g. a large number of database queries"